### ISSUE TITLE
Going to update for FS25?

### ISSUE THREAD

**Issue by ClaymoreRoomba** [Percedal/FS22_UpgradableFactories#53]
Just wondering if you were planning on updating the mod to FS25?

**Comment by seincraft** [comment]
Yeah please we need a update to FS25 can anyone do this for us =)?

**Comment by ToxxicAcex** [comment]
Please update it to FS25, this is the mod we all need

**Comment by MoneyNitch** [comment]
Shad0wlife has updated the mod to the FS25 version.
https://github.com/Shad0wlife/FS25_UpgradableFactories
